collective/auditlog/utils.py

```diff
@@ -39,23 +60,12 @@ def getUID(context):
     if uid is not None:
         return uid
 
-    if hasattr(context, "UID"):
-        return context.UID()
-
-    try:
-        return "/".join(context.getPhysicalPath())
-    except AttributeError:
-        pass
-
-    try:
-        return context.id
-    except AttributeError:
-        return "unknown"
+    uid = getattr(context, "UID", None)
+    return uid() if callable(uid) else uid
```

Comment:
This is missing the fallback in case UID is missing

Reply:
The idea is that it should return "None" instead of unknown to be consistent with the other return values.
path is already tracked seperately, so that would be a duplicate information.
Do we need the previous behavior because of data consistency?

Reply:
I do not know, I would just not change the existent behavior without a reason.